developer/src/kmcmplib/src/xstring.cpp

```diff
@@ -58,10 +58,10 @@ PKMX_WCHAR decxstr(PKMX_WCHAR p, PKMX_WCHAR pStart)
   p--;
   if(*p == UC_SENTINEL_EXTENDEDEND) {
     int n = 0;
-    while (p >= pStart && *p != UC_SENTINEL && n < 10) {
+    while (p > pStart && *p != UC_SENTINEL && n < 10) {
       p--; n++; }
```

Comment:
```suggestion
    while (p > pStart && *p != UC_SENTINEL && n < 10) {
      p--; 
      n++;
    }
```

Let's fix up formatting while at it.